apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,36 @@
+# Webserver Test Application
+
+### Features
+
+- HTTPS (based on mbedTLS h/w accelerate)
+- GET/PUT/POST/DELETE method
+- Supports Keep-alive
+- Chunked Transfer
+- Trailer header
+
+### Build Configuration
+
+> security_hal_test_tz
```

Comment:
The title is `Build Configuration` but the content is not matched I think. Could you check it?

Reply:
Made required changes